static/app/actionCreators/modal.tsx

```diff
@@ -447,3 +448,13 @@ export async function openTokenRegenerationConfirmationModal(options: ModalOptio
 
   openModal(deps => <Modal {...deps} {...options} />);
 }
+
+export async function openPrivateGamingSdkAccessModal(
```

Comment:
this function is unused, so I’d expect `knip` to complain here (which it doesn’t). I’ll look into that, but can we make the function used somewhere?